sensorlib/src/sensor/SemanticLidarSensor.py

```diff
@@ -153,6 +161,15 @@ def prefilter(self, detected_objects):
         # Filter by detected_object type Actor.type_id and Actor.semantic_tags are available for determining type;
         # semantic_tags effectively specifies the type of detected_object Possible types are listed in the CARLA
         # documentation: https://carla.readthedocs.io/en/0.9.10/ref_sensors/#semantic-segmentation-camera
+        print("Detected total objects of size: " + str(len(detected_objects)))
+        #for obj in detected_objects:
+        #    print("Detected object type: " + str(type(obj)))
+
+        detected_objects = list(
+            filter(lambda obj: obj is not None, detected_objects))
+        
+        print("After filtering errors size: " + str(len(detected_objects)))
```

Comment:
Do we want to use python's build in logging functionality instead of print statements directly so we can use:
- Log Levels
- Log Formatting 
- Possibly Log File creation.

Does not need to be addressed in this issue but I think is worth while to do for any python logic we use in deployment:
https://docs.python.org/3/library/logging.html

Reply:
removed all the print logs